index.html

```diff
@@ -0,0 +1,206 @@
+    </div>
+    <div class="p-div">
+        <p class="prg">
+            I can help you build a product ,
+             feature or website Look through some of my work and experience!
+              If you like what you see and have a project you need coded,
+               don’t hestiate to contact me.
+    
+        </p>
+    </div>
+        <hr class="hr1">  
+    </section>
+    <section>
+        <ul class="skills">
+            <li>
+                <h5 class="lang">Language</h5>
+                <ul class="tec">
+                    <li>JavaScript</li>
+                    <li>Ruby</li>
+                    <li>Html</li>
+                    <li>Css</li>
+                </ul>
+            </li>
+            <li>
+                <h5 class="lang">Frameworks</h5>
+                <ul class="tec">
+                    <li>Bootstrap</li>
+                    <li>Ruby on Rails</li>
+                    <li>RSPec</li>
+                    <li>CapyBara</li>
+                    <li>Selenium</li>
+                </ul>
+            </li>
+            <li>
+                <h5 class="lang">Skills</h5>
+                <ul class="tec">
+                    <li>Codekit</li>
+                    <li>Github</li>
+                    <li>Codepen</li>
+                    <li>Gitlab</li>
+                    <li>Terminal</li>
+                </ul>
+              </li> 
+        </ul>
+        <hr class="hr1">
+        <ul class="social-icon">
+            <li><img src="./images/GitHub.png" alt="github-icon"></li>
+            <li><img src="./images/Linkedin.png" alt="linkedin icon"></li>
+            <li><img src="./images/Angelist.png" alt="angel-icon"></li>
+            <li><img src="./images/Twitter.png" alt="Twitter-icon"></li>
+            <li><img src="./images/media.png" alt="m-icon"></li>
+        </ul> 
+        <div class="w-arrow">
+    <ul class="wk-arr">
+        <li><img src="./images/Group55.png" alt="down-arrow icon"></li>
+        <li>work</li>
+    </ul>
+        </div>
+    </section>
+
+</section>
+<main class="main-section">
+    <h2 class="r-work">MY RECENT WORK</h2>
+    <hr class="hr1">
+
+<ul>
+    <li class="bg1">
+
+        <img class="img-p" src="./images/Img-Placeholder.png" alt="img-placeholder">
+        
+        <h3 class="m-stories">Multi-Post Stories</h3>
+        <div>
+        <p class="c-prg">A daily selection of privately personalized reads;
+             no accounts or sign-ups required. has been the industry's
+             standard dummy text ever since the 1500s,
+              when an unknown printer took a standard dummy text.</p>
+        
+        <ul class="c-list">
+            <li>css</li>
+            <li class="c-d">|</li>
+            <li>html</li>
+            <li class="c-d">|</li>
+            <li>Bootstrap</li>
+            <li class="c-d">|</li>
+            <li>ruby</li>
+        </ul>
+        
+        
+        <button type="button" class="btn">See Project</button>
+        </div>
```

Comment:
- [ ] The button (e.g., "See project") should follow the button styling: Active, Hover, and Pressed (see design). Please work on the other "See project" buttons also.

![b1](https://user-images.githubusercontent.com/70483835/164933888-01385571-b30b-48f2-bbf3-be5d582b18b5.png)